INSTALL.md

```diff
@@ -854,6 +854,11 @@ This option will only produce a compliant FIPS provider if you have:
 
 Failure to do all of these will produce a non-compliant FIPS provider.
 
+### disable-fips186-4-ffc
+
+Disable support for legacy-only FIPS 186-4 FFC parameters in the FIPS
```

Comment:
This is specifically related to DH keys which should be stated here..

Reply:
well, in other cases these are removed already too, thus this is the last use of them, thus yes it is removing to DH keys; but also no other params like these are left in the FIPS provider.

I will mention DH keys, but will expand to say that it is also the last user of such params in the fips provider.